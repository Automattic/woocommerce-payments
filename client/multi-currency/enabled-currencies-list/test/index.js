/**
 * External dependencies
 */
import React from 'react';
import { cleanup, fireEvent, render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';

/**
 * Internal dependencies
 */
import EnabledCurrencies from '../';
import {
	useCurrencies,
	useAvailableCurrencies,
	useDefaultCurrency,
	useEnabledCurrencies,
} from 'wcpay/data';

jest.mock( 'wcpay/data', () => ( {
	useCurrencies: jest.fn(),
	useAvailableCurrencies: jest.fn(),
	useDefaultCurrency: jest.fn(),
	useEnabledCurrencies: jest.fn(),
} ) );

const availableCurrencies = {
	USD: {
		code: 'USD',
		rate: 1,
		name: 'United States (US) dollar',
		id: 'usd',
		is_default: true,
		flag: 'ðŸ‡ºðŸ‡¸',
		symbol: '$',
	},
	CAD: {
		code: 'CAD',
		rate: '1.206823',
		name: 'Canadian dollar',
		id: 'cad',
		is_default: false,
		flag: 'ðŸ‡¨ðŸ‡¦',
		symbol: '$',
	},
	GBP: {
		code: 'GBP',
		rate: '0.708099',
		name: 'Pound sterling',
		id: 'gbp',
		is_default: false,
		flag: 'ðŸ‡¬ðŸ‡§',
		symbol: 'Â£',
	},
	EUR: {
		code: 'EUR',
		rate: '0.826381',
		name: 'Euro',
		id: 'eur',
		is_default: false,
		flag: 'ðŸ‡ªðŸ‡º',
		symbol: 'â‚¬',
	},
	AED: {
		code: 'AED',
		rate: '3.6732',
		name: 'United Arab Emirates dirham',
		id: 'aed',
		is_default: false,
		flag: 'ðŸ‡¦ðŸ‡ª',
		symbol: 'Ø¯.Ø¥',
	},
	CDF: {
		code: 'CDF',
		rate: '2000',
		name: 'Congolese franc',
		id: 'cdf',
		is_default: false,
		flag: 'ðŸ‡¨ðŸ‡©',
		symbol: 'Fr',
	},
	NZD: {
		code: 'NZD',
		rate: '1.387163',
		name: 'New Zealand dollar',
		id: 'nzd',
		is_default: false,
		flag: 'ðŸ‡³ðŸ‡¿',
		symbol: '$',
	},
	DKK: {
		code: 'DKK',
		rate: '6.144615',
		name: 'Danish krone',
		id: 'dkk',
		is_default: false,
		flag: 'ðŸ‡©ðŸ‡°',
		symbol: 'DKK',
	},
	BIF: {
		code: 'BIF',
		rate: '1974',
		name: 'Burundian franc',
		id: 'bif',
		is_default: false,
		flag: 'ðŸ‡§ðŸ‡®',
		symbol: 'Fr',
	},
	CLP: {
		code: 'CLP',
		rate: '706.8',
		name: 'Chilean peso',
		id: 'clp',
		is_default: false,
		flag: 'ðŸ‡¨ðŸ‡±',
		symbol: '$',
	},
};

const enabledCurrencies = {
	USD: {
		code: 'USD',
		rate: 1,
		name: 'United States (US) dollar',
		id: 'usd',
		is_default: true,
		flag: 'ðŸ‡ºðŸ‡¸',
		symbol: '$',
	},
	CAD: {
		code: 'CAD',
		rate: '1.42',
		name: 'Canadian dollar',
		id: 'cad',
		is_default: false,
		flag: 'ðŸ‡¨ðŸ‡¦',
		symbol: '$',
	},
	GBP: {
		code: 'GBP',
		rate: '0.708099',
		name: 'Pound sterling',
		id: 'gbp',
		is_default: false,
		flag: 'ðŸ‡¬ðŸ‡§',
		symbol: 'Â£',
	},
	EUR: {
		code: 'EUR',
		rate: '0.826381',
		name: 'Euro',
		id: 'eur',
		is_default: false,
		flag: 'ðŸ‡ªðŸ‡º',
		symbol: 'â‚¬',
	},
};

const defaultCurrency = {
	code: 'USD',
	rate: 1,
	name: 'United States (US) dollar',
	id: 'usd',
	is_default: true,
	flag: 'ðŸ‡ºðŸ‡¸',
	symbol: '$',
};

useCurrencies.mockReturnValue( {
	currencies: {
		available: availableCurrencies,
		enabled: enabledCurrencies,
		default: defaultCurrency,
	},
	isLoading: false,
} );

useAvailableCurrencies.mockReturnValue( availableCurrencies );

useDefaultCurrency.mockReturnValue( defaultCurrency );

useEnabledCurrencies.mockReturnValue( {
	enabledCurrencies: enabledCurrencies,
	submitEnabledCurrenciesUpdate: () => {},
} );

describe( 'Multi Currency enabled currencies list', () => {
	beforeEach( () => {
		jest.clearAllMocks();
		global.wcpaySettings = { zeroDecimalCurrencies: [] };
	} );

	test( 'Enabled currencies list renders correctly', () => {
		const { container } = render( <EnabledCurrencies /> );
		expect( container ).toMatchSnapshot();
	} );

	test( 'Available currencies modal renders correctly', () => {
		render( <EnabledCurrencies /> );
		expect(
			screen.queryByRole( 'dialog', { name: /add enabled currencies/i } )
		).not.toBeInTheDocument();
		fireEvent.click(
			screen.getByRole( 'button', { name: /add currencies/i } )
		);
		const modal = screen.queryByRole( 'dialog', {
			name: /add enabled currencies/i,
		} );
		expect( modal ).toBeInTheDocument();
		expect( modal ).toMatchSnapshot();
	} );

	test( 'Remove currency modal renders correctly', () => {
		render( <EnabledCurrencies /> );
		expect(
			screen.queryByRole( 'dialog', { name: /remove euro/i } )
		).not.toBeInTheDocument();
		fireEvent.click(
			screen.getByRole( 'button', {
				name: /remove euro as an enabled currency/i,
			} )
		);
		const modal = screen.queryByRole( 'dialog', { name: /remove euro/i } );
		expect( modal ).toBeInTheDocument();
		expect( modal ).toMatchSnapshot();
	} );

	test( 'Modal should clear search term on cancel and update selected', () => {
		for ( const name of [ /cancel/i, /update selected/i ] ) {
			render( <EnabledCurrencies /> );
			userEvent.click(
				screen.getByRole( 'button', {
					name: /add currencies/i,
				} )
			);
			userEvent.type( screen.getByRole( 'textbox' ), 'dollar' );
			userEvent.click(
				screen.getByRole( 'button', {
					name,
				} )
			);
			userEvent.click(
				screen.getByRole( 'button', {
					name: /add currencies/i,
				} )
			);
			expect(
				screen.queryByDisplayValue( 'dollar' )
			).not.toBeInTheDocument();
			cleanup();
		}
	} );
} );
