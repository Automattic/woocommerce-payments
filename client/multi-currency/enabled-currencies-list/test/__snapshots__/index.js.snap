// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Multi Currency enabled currencies list Available currencies modal renders correctly 1`] = `
<div
  aria-labelledby="components-modal-header-0"
  class="components-modal__frame add-enabled-currencies-modal"
  role="dialog"
  tabindex="-1"
>
  <div
    class="components-modal__content"
    role="document"
  >
    <div
      class="components-modal__header"
    >
      <div
        class="components-modal__header-heading-container"
      >
        <h1
          class="components-modal__header-heading"
          id="components-modal-header-0"
        >
          Add enabled currencies
        </h1>
      </div>
      <button
        aria-label="Close dialog"
        class="components-button has-icon"
        type="button"
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="24"
          role="img"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"
          />
        </svg>
      </button>
    </div>
    <div
      class="add-enabled-currencies-modal__search"
    >
      <div
        class="search"
      >
        <svg
          aria-hidden="true"
          class="search__icon"
          focusable="false"
          height="24"
          role="img"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M13.5 6C10.5 6 8 8.5 8 11.5c0 1.1.3 2.1.9 3l-3.4 3 1 1.1 3.4-2.9c1 .9 2.2 1.4 3.6 1.4 3 0 5.5-2.5 5.5-5.5C19 8.5 16.5 6 13.5 6zm0 9.5c-2.2 0-4-1.8-4-4s1.8-4 4-4 4 1.8 4 4-1.8 4-4 4z"
          />
        </svg>
        <input
          class="components-text-control__input"
          placeholder="Search currencies"
          type="text"
          value=""
        />
      </div>
    </div>
    <h3>
      All currencies
    </h3>
    <div
      class="add-enabled-currencies-modal__content"
    >
      <ul>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  checked=""
                  class="components-checkbox-control__input"
                  code="CAD"
                  id="inspector-checkbox-control-0"
                  type="checkbox"
                  value="1"
                />
                <svg
                  aria-hidden="true"
                  class="components-checkbox-control__checked"
                  focusable="false"
                  height="24"
                  role="img"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M18.3 5.6L9.9 16.9l-4.6-3.4-.9 1.2 5.8 4.3 9.3-12.6z"
                  />
                </svg>
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-0"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡¨ðŸ‡¦
                </span>
                 
                <span>
                  Canadian dollar
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  $
                   CAD
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  checked=""
                  class="components-checkbox-control__input"
                  code="GBP"
                  id="inspector-checkbox-control-1"
                  type="checkbox"
                  value="1"
                />
                <svg
                  aria-hidden="true"
                  class="components-checkbox-control__checked"
                  focusable="false"
                  height="24"
                  role="img"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M18.3 5.6L9.9 16.9l-4.6-3.4-.9 1.2 5.8 4.3 9.3-12.6z"
                  />
                </svg>
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-1"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡¬ðŸ‡§
                </span>
                 
                <span>
                  Pound sterling
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  Â£
                   GBP
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  checked=""
                  class="components-checkbox-control__input"
                  code="EUR"
                  id="inspector-checkbox-control-2"
                  type="checkbox"
                  value="1"
                />
                <svg
                  aria-hidden="true"
                  class="components-checkbox-control__checked"
                  focusable="false"
                  height="24"
                  role="img"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M18.3 5.6L9.9 16.9l-4.6-3.4-.9 1.2 5.8 4.3 9.3-12.6z"
                  />
                </svg>
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-2"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡ªðŸ‡º
                </span>
                 
                <span>
                  Euro
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  â‚¬
                   EUR
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="AED"
                  id="inspector-checkbox-control-3"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-3"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡¦ðŸ‡ª
                </span>
                 
                <span>
                  United Arab Emirates dirham
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  Ø¯.Ø¥
                   AED
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="CDF"
                  id="inspector-checkbox-control-4"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-4"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡¨ðŸ‡©
                </span>
                 
                <span>
                  Congolese franc
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  Fr
                   CDF
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="NZD"
                  id="inspector-checkbox-control-5"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-5"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡³ðŸ‡¿
                </span>
                 
                <span>
                  New Zealand dollar
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  $
                   NZD
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="DKK"
                  id="inspector-checkbox-control-6"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-6"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡©ðŸ‡°
                </span>
                 
                <span>
                  Danish krone
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  DKK
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="BIF"
                  id="inspector-checkbox-control-7"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-7"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡§ðŸ‡®
                </span>
                 
                <span>
                  Burundian franc
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  Fr
                   BIF
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
        <li
          class="enabled-currency-checkbox"
        >
          <div
            class="components-base-control css-wdf2ti-Wrapper e1puf3u0"
          >
            <div
              class="components-base-control__field css-11vcxb9-StyledField e1puf3u1"
            >
              <span
                class="components-checkbox-control__input-container"
              >
                <input
                  class="components-checkbox-control__input"
                  code="CLP"
                  id="inspector-checkbox-control-8"
                  type="checkbox"
                  value="1"
                />
              </span>
              <label
                class="components-checkbox-control__label"
                for="inspector-checkbox-control-8"
              >
                <span
                  class="enabled-currency-checkbox__flag"
                >
                  ðŸ‡¨ðŸ‡±
                </span>
                 
                <span>
                  Chilean peso
                </span>
                 
                <span
                  class="enabled-currency-checkbox__code"
                >
                  (
                  $
                   CLP
                  )
                </span>
              </label>
            </div>
          </div>
        </li>
      </ul>
    </div>
    <div
      class="add-enabled-currencies-modal__footer"
    >
      <button
        class="components-button is-secondary"
        type="button"
      >
        Cancel
      </button>
      <button
        class="components-button is-primary"
        type="button"
      >
        Update selected
      </button>
    </div>
  </div>
</div>
`;

exports[`Multi Currency enabled currencies list Enabled currencies list renders correctly 1`] = `
<div>
  <div
    class="components-card is-size-medium wcpay-multi-currency__enabled-currencies css-1xs3c37-CardUI e1q7k77g0"
  >
    <div
      class="components-card__body is-size-medium wcpay-multi-currency__enabled-currencies-header css-xmjzce-BodyUI e1q7k77g3"
    >
      <div>
        Enabled Currencies
      </div>
      <div>
        Exchange rate
      </div>
    </div>
    <hr
      class="components-card__divider css-lh2anh-DividerUI e1q7k77g5"
      role="separator"
    />
    <div
      class="components-card__body css-xmjzce-BodyUI e1q7k77g3"
    >
      <ul
        class="enabled-currencies-list"
      >
        <li
          class="enabled-currency usd"
        >
          <div
            class="enabled-currency__container"
          >
            <div
              class="enabled-currency__flag"
            >
              ðŸ‡ºðŸ‡¸
            </div>
            <div
              class="enabled-currency__label"
            >
              United States (US) dollar
            </div>
            <div
              class="enabled-currency__code"
            >
              (
              $
               
              USD
              )
            </div>
          </div>
          <div
            class="enabled-currency__rate"
          >
            Default currency
          </div>
          <div
            class="enabled-currency__actions"
          >
            <a
              aria-label="Edit United States (US) dollar"
              class="components-button enabled-currency__action edit is-link"
              href="admin.php?page=wc-settings&tab=wcpay_multi_currency&section=usd"
            >
              <span
                class="dashicon dashicons dashicons-edit"
              />
            </a>
          </div>
        </li>
        <li
          class="enabled-currency cad"
        >
          <div
            class="enabled-currency__container"
          >
            <div
              class="enabled-currency__flag"
            >
              ðŸ‡¨ðŸ‡¦
            </div>
            <div
              class="enabled-currency__label"
            >
              Canadian dollar
            </div>
            <div
              class="enabled-currency__code"
            >
              (
              $
               
              CAD
              )
            </div>
          </div>
          <div
            class="enabled-currency__rate"
          >
            1 USD â†’ 1.42 CAD
          </div>
          <div
            class="enabled-currency__actions"
          >
            <a
              aria-label="Edit Canadian dollar"
              class="components-button enabled-currency__action edit is-link"
              href="admin.php?page=wc-settings&tab=wcpay_multi_currency&section=cad"
            >
              <span
                class="dashicon dashicons dashicons-edit"
              />
            </a>
            <button
              aria-label="Remove Canadian dollar as an enabled currency"
              class="components-button enabled-currency__action delete is-link"
              type="button"
            >
              <span
                class="dashicon dashicons dashicons-trash"
              />
            </button>
          </div>
        </li>
        <li
          class="enabled-currency gbp"
        >
          <div
            class="enabled-currency__container"
          >
            <div
              class="enabled-currency__flag"
            >
              ðŸ‡¬ðŸ‡§
            </div>
            <div
              class="enabled-currency__label"
            >
              Pound sterling
            </div>
            <div
              class="enabled-currency__code"
            >
              (
              Â£
               
              GBP
              )
            </div>
          </div>
          <div
            class="enabled-currency__rate"
          >
            1 USD â†’ 0.71 GBP
          </div>
          <div
            class="enabled-currency__actions"
          >
            <a
              aria-label="Edit Pound sterling"
              class="components-button enabled-currency__action edit is-link"
              href="admin.php?page=wc-settings&tab=wcpay_multi_currency&section=gbp"
            >
              <span
                class="dashicon dashicons dashicons-edit"
              />
            </a>
            <button
              aria-label="Remove Pound sterling as an enabled currency"
              class="components-button enabled-currency__action delete is-link"
              type="button"
            >
              <span
                class="dashicon dashicons dashicons-trash"
              />
            </button>
          </div>
        </li>
        <li
          class="enabled-currency eur"
        >
          <div
            class="enabled-currency__container"
          >
            <div
              class="enabled-currency__flag"
            >
              ðŸ‡ªðŸ‡º
            </div>
            <div
              class="enabled-currency__label"
            >
              Euro
            </div>
            <div
              class="enabled-currency__code"
            >
              (
              â‚¬
               
              EUR
              )
            </div>
          </div>
          <div
            class="enabled-currency__rate"
          >
            1 USD â†’ 0.83 EUR
          </div>
          <div
            class="enabled-currency__actions"
          >
            <a
              aria-label="Edit Euro"
              class="components-button enabled-currency__action edit is-link"
              href="admin.php?page=wc-settings&tab=wcpay_multi_currency&section=eur"
            >
              <span
                class="dashicon dashicons dashicons-edit"
              />
            </a>
            <button
              aria-label="Remove Euro as an enabled currency"
              class="components-button enabled-currency__action delete is-link"
              type="button"
            >
              <span
                class="dashicon dashicons dashicons-trash"
              />
            </button>
          </div>
        </li>
      </ul>
    </div>
    <hr
      class="components-card__divider css-lh2anh-DividerUI e1q7k77g5"
      role="separator"
    />
    <div
      class="components-card__body is-size-medium wcpay-multi-currency__available-currencies-container css-xmjzce-BodyUI e1q7k77g3"
    >
      <button
        class="components-button wcpay-multi-currency__available-currencies-modal is-secondary"
        type="button"
      >
        Add currencies
      </button>
    </div>
  </div>
</div>
`;

exports[`Multi Currency enabled currencies list Remove currency modal renders correctly 1`] = `
<div
  aria-labelledby="components-modal-header-1"
  class="components-modal__frame enabled-currency-delete-modal"
  role="dialog"
  tabindex="-1"
>
  <div
    class="components-modal__content"
    role="document"
  >
    <div
      class="components-modal__header"
    >
      <div
        class="components-modal__header-heading-container"
      >
        <h1
          class="components-modal__header-heading"
          id="components-modal-header-1"
        >
          Remove Euro
        </h1>
      </div>
      <button
        aria-label="Close dialog"
        class="components-button has-icon"
        type="button"
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="24"
          role="img"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"
          />
        </svg>
      </button>
    </div>
    <p>
      Please confirm you would like to remove 
      <strong>
        Euro
      </strong>
       as an enabled currency in your store.
    </p>
    <div
      class="enabled-currency-delete-modal__footer"
    >
      <button
        class="components-button is-secondary"
        type="button"
      >
        Cancel
      </button>
      <button
        class="components-button is-primary is-destructive"
        type="button"
      >
        Remove currency
      </button>
    </div>
  </div>
</div>
`;
